feature-libs/product-configurator/rulebased/core/facade/configurator-commons.service.ts

```diff
@@ -112,8 +112,9 @@ export class ConfiguratorCommonsService {
    * @param changedAttribute - Changes attribute
    */
   updateConfiguration(
+    changedAttribute: Configurator.Attribute,
     ownerKey: string,
-    changedAttribute: Configurator.Attribute
+    updateType: Configurator.UpdateType
```

Comment:
breaking change - change the order of method params

Reply:
It was done!